scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime, timedelta
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+m = DuplicateModel()
+m.train()
+
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+delta = timedelta(days=21)
+test_bugs = bugzilla.download_bugs_between(datetime.now() - delta, datetime.now())
+
+test_bugs = [bug for bug in test_bugs if not bug["creator"] in REPORTERS_TO_IGNORE]
+
+bug_tuples = itertools.combinations(test_bugs, 2)
+probs = [m.classify(bug_tuple)[0] for bug_tuple in bug_tuples]
```

Comment:
You are still doing the classification bug couple per bug couple. It makes way more sense to classify them all together, as it's way faster.

Reply:
https://github.com/mozilla/bugbug/pull/548#discussion_r290684822